### DISCUSSION TITLE
Whether to display both left and right channels at the same time？

### DISCUSSION THREAD

**Discussion by hotYan** [katspaugh/wavesurfer.js#3797] [Q&A] [answered]
<img width="1419" alt="image" src="https://github.com/user-attachments/assets/20775807-cf10-431c-85fe-76311d909198">

**Comment by katspaugh** [comment]
Sorry, I don't understand the question.

**Reply by hotYan** [reply]
Whether to show left and right sound track as the same time？

**Reply by hotYan** [reply]
Can this show waveforms for 2-channel stereo? Like when the right earphone and left earphone is playing different sound in the same time whether this can show two waveforms?

**Reply by katspaugh** [reply, marked as answer]
Yes, there's a splitChannels option: https://wavesurfer.xyz/examples/?02daa0b4592c3120d333872fd176f04f
